### PR TITLE
Add escape and click off modal close modal

### PR DESCRIPTION
**Pull request by dtriley6**

### CODE REVIEW

File: dist/ti-modal.js [hunk 1]
```diff
@@ -108,6 +110,20 @@ define("ti-modal",
     });
 
     ModalComponent = Ember.Component.extend({
+      bindClosingBehavior: function() {
+        var modalComponent = this;
+        
+        $(document).bind('keyup.modal', function(e) {
```
Comment (chrisvariety) [inline dist/ti-modal.js]: `As of jQuery 1.7, the .on() method is the preferred method for attaching event handlers to a document.` from jq docs.

You could also use `$.proxy` to keep the `this` scope and avoid having to define `modalComponent` entirely.

Comment (dtriley6) [inline dist/ti-modal.js]: Oh right, forgot about `on` and `off` and I'll fix up with `$.proxy`.

**Comment by dtriley6** [comment]
@chrisvariety, made those updates for jQuery and bumped version to `2.0.2`.

File: dist/ti-modal.js [hunk 2]
```diff
@@ -144,3 +83,78 @@ define("ti-modal/initializer",
       });
     });
   });
+define("templates/ti-modal", function () { return Ember.TEMPLATES["ti-modal"] = Ember.Handlebars.template(function anonymous(Handlebars,depth0,helpers,partials,data
```
Comment (chrisvariety) [inline dist/ti-modal.js]: I think this is wrong, needs to be `define("templates/ti-modal", ["exports"], function (__exports__) { __exports__["default"] = Ember.Handlebars.template(...`

Comment (dtriley6) [inline dist/ti-modal.js]: woops, missed that on the last `gulp`

Comment (dtriley6) [inline dist/ti-modal.js]: actually, did it, but didn't save the file.

**Pull request merged**